components/mosart/src/riverroute/RtmMod.F90

```diff
@@ -1790,12 +1803,24 @@ subroutine Rtmini(rtm_active,flood_active)
 
     call t_stopf('mosarti_mosart_init')
 
+    call t_startf('mosarti_sediment_init')
```

Comment:
Move t_startf and t_stop into the if block

Reply:
Done